src/components/PeopleTable.tsx

```diff
@@ -1,683 +1,171 @@
-            </a>
-          </td>
-          <td>
-            <a href="#/people/carel-haverbeke-1796">Carel Haverbeke</a>
-          </td>
-        </tr>
-
-        <tr data-cy="person">
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/maria-sturm-1835"
-            >
-              Maria Sturm
-            </a>
-          </td>
-          <td>f</td>
-          <td>1835</td>
-          <td>1917</td>
-          <td>Seraphina Spelier</td>
-          <td>Charles Sturm</td>
-        </tr>
-
-        <tr data-cy="person">
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/emma-de-milliano-1876"
-            >
-              Emma de Milliano
-            </a>
-          </td>
-          <td>f</td>
-          <td>1876</td>
-          <td>1956</td>
-          <td>Sophia van Damme</td>
-          <td>Petrus de Milliano</td>
-        </tr>
-
-        <tr data-cy="person">
-          <td>
-            <a href="#/people/emile-haverbeke-1877">Emile Haverbeke</a>
-          </td>
-          <td>m</td>
-          <td>1877</td>
-          <td>1968</td>
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/maria-sturm-1835"
-            >
-              Maria Sturm
-            </a>
-          </td>
-          <td>
-            <a href="#/people/carolus-haverbeke-1832">Carolus Haverbeke</a>
-          </td>
-        </tr>
-      </tbody>
-    </table>
+              </th>
+            ))}
+
+            <th>Mother</th>
+            <th>Father</th>
+          </tr>
+        </thead>
+        <tbody>
+          {sortedPeople.map((person) => {
+            const {
+              name, sex, born, died, fatherName, motherName, slug,
+            } = person;
+
+            const selectedMother = people.find(el => {
+              return el.name === motherName;
+            });
+
+            const selectedFather = people.find(el => {
+              return el.name === fatherName;
+            });
+
+            return (
+              <tr
+                data-cy="person"
+                key={slug}
+                className={classNames(
+                  { 'has-background-warning': slug === selectedSlug },
+                )}
+              >
+                <td>
+                  <PersonLink
+                    slug={slug}
+                    sex={sex}
+                    name={name}
+                  />
+                </td>
+                <td>{sex}</td>
+                <td>{born}</td>
+                <td>{died}</td>
+                <td>
+                  {selectedMother
+                    ? (
+                      <ParentLink
+                        sex={sex}
+                        parentName={motherName}
+                        selectedParent={selectedMother}
+                      />
+                    )
+                    : <div>{motherName || '-'}</div>}
```

Comment:
Check your code formatting

```suggestion
                  {selectedMother ? (
                      <ParentLink
                        sex={sex}
                        parentName={motherName}
                        selectedParent={selectedMother}
                      />
                    ) : (
                      <div>{motherName || '-'}</div>
                    )}
```